Gruntfile.js

```diff
@@ -953,10 +958,14 @@ module.exports = function(grunt) {
     'exec:pack-node-modules',
   ]);
 
-  grunt.registerTask('start-webdriver', 'Starts Protractor Webdriver', [
-    'replace:webdriver-version',
-    'exec:start-webdriver',
-  ]);
+  grunt.registerTask('start-webdriver', 'Starts Protractor Webdriver', function() {
+    grunt.task.run('replace:webdriver-version');
+    if (CI) {
+      grunt.task.run('exec:start-webdriver-ci');
+    } else {
+      grunt.task.run('exec:start-webdriver');
```

Comment:
On second though, why not just use the new `start_webdriver` script for non-ci testing as well? It would save the heartache of when local chrome is not up to date :)

Reply:
I think we can do that but there's a bit more work to do OS checking to then find the chrome executable. Since we know that CI runs in ubuntu the `google-chrome-stable` will work. Where as on Mac and windows that command does nothing :-(

Reply:
:(